substrate/frame/revive/src/exec.rs

```diff
@@ -885,9 +885,9 @@ where
 			args,
 			value,
 			gas_meter,
-			Weight::zero(),
+			Weight::max_value(),
 			storage_meter,
-			BalanceOf::<T>::zero(),
+			BalanceOf::<T>::max_value(),
```

Comment:
The first frame is also created by `nested` and will be limited to 63/64. We need to make sure the first one can use the full resources.

Reply:
>We need to make sure the first one can use the full resources.

I did not understand; may you elaborate on what exactly that would mean here?

Reply:
In the current code the overall transaction will be limited to 63/64. We don't want this. We only want sub calls to be limited. But not the first contract called by the externally owned account. That one should be allowed to use all the gas.